expressions/methods.md

```diff
@@ -228,6 +228,47 @@ primitive Foo
     f.>add_option(o1).>add_option(o2).make_object() // Works. The expression returns an Object
 ```
 
+## Behaviours
+Behaviours are asynchronous method calls. As such, they cannot return anything, and the caller will not wait for the call to finish. Here is a quick example to illustrate this certainly :
+
+```pony
+/** 
+This program will certainly print the following :
```

Comment:
I am not 100% sure that this is always the case. afaik it could be that `OtherActor.ping` is called before the last print in `Main.create` is evaluated. But i can be wrong about this.